lib/urldata.h

```diff
@@ -1861,6 +1860,7 @@ struct Curl_easy {
   struct Curl_easy *next;
   struct Curl_easy *prev;
 
+  int retrycount; /* number of retries on a new connection */
```

Comment:
I would like this to get added to the `struct UrlState` sub struct instead, which we generally use for these kind of state variables.

Reply:
Agreed. Done.